recipes/idaes-pse/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# idaes_pse/meta.yaml
+{% set name = "idaes-pse" %}
+{% set version = "2.6.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  {# verify sha using curl -sL $URL | sha256sum #}
+  url: https://github.com/IDAES/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dcc09ff01b30d5fc60cf0b4fc68b41b0c763ea8b6ae78237f5d4da3f563bb334
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - idaes = idaes.commands.base:command_base
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-cache-dir --ignore-installed"
+
+requirements:
+  host:
+    - python {{ python_min }}
```

Comment:
Here are way too much things listed. There should only be the things that are required to build the package, e.g. `setuptools`.

Reply:
Thanks, I see what you mean. Done in ed160f7b338e34180b11e19210c8036f10c231af